Fixes the following on macOS 11 and below:

In file included from /opt/local/var/macports/build/_opt_bblocal_var_buildworker_ports_build_ports_lang_vlang/vlang/work/vc-fed3800f83b51aba951a88e4d3d9d37ed5c10ee2/v.c:1943:
/Library/Developer/CommandLineTools/SDKs/MacOSX11.sdk/usr/include/sys/ptrace.h:99:42: error: unknown type name 'caddr_t'
int     ptrace(int _request, pid_t _pid, caddr_t _addr, int _data);
                                         ^
1 error generated.
make: *** [all] Error 1

--- vc/v.c.original	2023-03-21 13:52:00.000000000 +0000
+++ vc/v.c	2023-03-21 13:52:38.000000000 +0000
@@ -1945,6 +1945,7 @@
 #if defined(__has_include)
 
 #if __has_include(<sys/ptrace.h>)
+#include <sys/types.h>
 #include <sys/ptrace.h>
 #else
 #error VERROR_MESSAGE Header file <sys/ptrace.h>, needed for module `os` was not found. Please install the corresponding development headers.
